modules/server/src/graphqlRoutes.js

```diff
@@ -281,9 +281,19 @@ export const createSchemasFromConfigs = async ({
 		/**
 		 * Federated Network Search
 		 */
-		const { networkSchema } = await createSchemaFromNetworkConfig({
-			networkConfigs: configsFromFiles[ConfigProperties.NETWORK_AGGREGATION],
-		});
+		if (ENABLE_NETWORK_AGGREGATION) {
+			const { networkSchema } = await createSchemaFromNetworkConfig({
+				networkConfigs: configsFromFiles[ConfigProperties.NETWORK_AGGREGATION].map((config) => ({
+					...config,
+					/**
+					 * part of the gql schema is generated dynamically
+					 * in the case of the "file" field, the field name and type name are the same
+					 * it's more flexible to define it here as an additional property than to confuse functions further down the pipeline
+					 */
+					documentName: config.documentType,
```

Comment:
I'd be inclined to define networkConfigs as a variable first

```const networkConfigs = configsFromFiles[ConfigProperties.NETWORK_AGGREGATION].map((config) => ({
					...config,
					/**
					 * part of the gql schema is generated dynamically
					 * in the case of the "file" field, the field name and type name are the same
					 * it's more flexible to define it here as an additional property than to confuse functions further down the pipeline
					 */
					documentName: config.documentType}));
const { networkSchema } = await createSchemaFromNetworkConfig({ networkConfigs });```

Reply:
interesting. for what reason?
I'm mostly trying to avoid doing:
```
const networkConfigs = configsFromFile......
const adjustedNetworkConfigs = networkConfigs....
``

Reply:
Mainly readability, and generally to reduce logic in arguments.

If `createSchemaFromNetworkConfig` arguments ever change it becomes a lot cleaner passing in variable object keys.

If there's ever an error it's easy to trace back to the original definition.